### PR TITLE
feat/MET-4720_MET-4761 add skos notation date normalization data to mongo

### PR DESCRIPTION
**Pull request by jeortizquan**
**remark**: we will need to wait until corelib makes their official release. The pom needs to run with 2.15.8. At the moment  works with 2.15.8-SNAPSHOT version of corelib.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -137,7 +137,7 @@
     <version.com.zoho.crm.java-sdk>3.0.1</version.com.zoho.crm.java-sdk>
     <version.apache.commons.text>1.9</version.apache.commons.text>
     <version.commonscompress>1.21</version.commonscompress>
-    <version.corelib>2.15.3</version.corelib>
+    <version.corelib>2.15.8</version.corelib>
```
Comment (jochen-vermeulen) [inline pom.xml:140]: I suppose that for now we'll have to use the snapshot version. Right now the build is broken.

File: metis-indexing/src/main/java/eu/europeana/indexing/mongo/TimespanUpdater.java [hunk 2]
```diff
@@ -23,5 +23,6 @@ protected void update(MongoPropertyUpdater<TimespanImpl> propertyUpdater) {
     propertyUpdater.updateMap("isPartOf", TimespanImpl::getIsPartOf);
     propertyUpdater.updateMap("dctermsHasPart", TimespanImpl::getDctermsHasPart);
     propertyUpdater.updateArray("owlSameAs", TimespanImpl::getOwlSameAs);
+    propertyUpdater.updateMap("notation", TimespanImpl::getSkosNotation);
```
Comment (jochen-vermeulen) [inline metis-indexing/src/main/java/eu/europeana/indexing/mongo/TimespanUpdater.java:26]: In order to get the skos notation from `TimespanImpl` you will first need to set it. See `TimespanFieldInput`.

File: pom.xml (hunk 1 above)
Comment (jeortizquan) [reply pom.xml:140]: Updated to the snapshot version.

File: metis-indexing/src/main/java/eu/europeana/indexing/mongo/TimespanUpdater.java (hunk 2 above)
Comment (jeortizquan) [reply metis-indexing/src/main/java/eu/europeana/indexing/mongo/TimespanUpdater.java:26]: I see. Updated the ```TimespanFieldInput``` class

**Pull request merged**